test/coupon_code_test.rb

```diff
@@ -52,11 +52,11 @@ def generate(*args)
     end
 
     it 'should accept a short code with correct parts.' do
-      CouponCode.validate('1K7Q-CTFM', 2).wont_be_nil
+      CouponCode.validate('1K7Q-CTFM', parts: 2).wont_be_nil
     end
 
     it 'should reject a short code with wrong parts.' do
-      CouponCode.validate('CTFM-1K7Q', 2).must_be_nil
+      CouponCode.validate('CTFM-1K7Q', parts: 2).must_be_nil
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.